server/services/clean-up.js

```diff
@@ -57,9 +57,130 @@ const cleanStaleInProgressParticipant = async () => {
   return results;
 };
 
+/**
+ * @description - Creates a table of all participants that should currently be Open - Never engaged, or current status = rejected / reject_ack
+ * @returns {Promise<Array>}
+ */
+const createOpenParticipantsTable = async () => {
```

Comment:
is `openParticipantsTable` the right term for this?

Reply:
Nope, outdated when I renamed / reworked everything, thanks